python/mujinwebstackclient/webstackclient.py

```diff
@@ -78,6 +78,78 @@ def _FormatHTTPDate(dt):
     month = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'][dt.month - 1]
     return '%s, %02d %s %04d %02d:%02d:%02d GMT' % (weekday, dt.day, month, dt.year, dt.hour, dt.minute, dt.second)
 
+def BreakLargeQuery(queryFunction):
```

Comment:
We can use the iterator inside this decorator to prevent repeated logic

Reply:
hmmm... 
Then it's hard to return `totalCount` to the user. I don't see an elegant way to achieve this after reuse iterator. Any suggestion? Also, I think it will be slightly slower if we reuse the iterator.